scripts/lint_terraform.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script formats, validates, and lints the terraform modules in the repo.
+# It is intended to be run from the root of the repo.
+
+set -e
+
+echo "##### Installing tflint tool #####"
```

Comment:
I think this should probably be python, its pushing the limit of what we should be doing with bash